project.clj

```diff
@@ -44,21 +36,20 @@
                  ;; We are using an exlusion here because logback dependencies should
                  ;; be inherited from trapperkeeper to avoid accidentally bringing
                  ;; in different versions of the three different logback artifacts
-                 [net.logstash.logback/logstash-logback-encoder "4.5.1" :exclusions [ch.qos.logback/logback-access
-                                                                                     ch.qos.logback/logback-core]]
-
-                 [puppetlabs/jruby-utils "0.2.1"]
-                 [puppetlabs/trapperkeeper ~tk-version]
-                 [puppetlabs/trapperkeeper-authorization "0.7.0"]
-                 [puppetlabs/trapperkeeper-scheduler "0.0.1"]
-                 [puppetlabs/trapperkeeper-status "0.3.5"]
-                 [puppetlabs/kitchensink ~ks-version]
-                 [puppetlabs/ssl-utils "0.8.1"]
-                 [puppetlabs/ring-middleware "1.0.0"]
-                 [puppetlabs/dujour-version-check "0.1.2" :exclusions [org.clojure/tools.logging]]
-                 [puppetlabs/http-client "0.5.0"]
-                 [puppetlabs/comidi "0.3.1"]
-                 [puppetlabs/i18n "0.4.3"]]
+                 [net.logstash.logback/logstash-logback-encoder "4.5.1"]
```

Comment:
I think it probably is worth putting in clj-parent.  I can do that.

I think managed dependencies would still trump the transitive dependencies you're referring to, though, so that shouldn't be an issue.